app/src/main/java/org/stepik/android/view/course_revenue/ui/adapter/delegate/CourseBenefitsAdapterDelegate.kt

```diff
@@ -47,15 +47,26 @@ class CourseBenefitsAdapterDelegate(
             decimalFormat.minimumFractionDigits = 2
 
             purchaseRefundIcon.setImageDrawable(getIconDrawable(data.courseBenefit))
-            purchaseRefundName.text = data.user?.fullName ?: data.courseBenefit.buyer.toString()
+            purchaseRefundName.text =
+                if (data.courseBenefit.buyer == null && !data.courseBenefit.isInvoicePayment) {
```

Comment:
Maybe add computable field for types?

Reply:
Added `isManual` property in `CourseBenefit`